pkg/tenant/api/v1/client_test.go

```diff
@@ -132,121 +132,83 @@ func TestStatus(t *testing.T) {
 	})
 }
 
-func TestUserList(t *testing.T) {
-	fixture := &api.UserPage{}
-	// Creates a test server
-	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		require.Equal(t, http.MethodGet, r.Method)
-		require.Equal(t, "/v1/users", r.URL.Path)
-
-		w.Header().Add("Content-Type", "application/json; charset=utf-8")
-		w.WriteHeader(http.StatusOK)
-		json.NewEncoder(w).Encode(fixture)
-
-	}))
-	defer ts.Close()
-
-	// Creates a client to execute tests against the test server
-	client, err := api.New(ts.URL)
-	require.NoError(t, err)
-
-	out, err := client.UserList(context.Background(), &api.UserQuery{})
-	require.NoError(t, err)
-	require.Equal(t, fixture, out)
-
-}
-
-func TestUserCreate(t *testing.T) {
-	fixture := &api.User{
-		ID:       "001",
-		UserName: "username01",
-	}
-	// Creates a test server
-	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		require.Equal(t, http.MethodPost, r.Method)
-		require.Equal(t, "/v1/users", r.URL.Path)
-
-		w.Header().Add("Content-Type", "application/json; charset=utf-8")
-		w.WriteHeader(http.StatusOK)
-		json.NewEncoder(w).Encode(fixture)
-
-	}))
-	defer ts.Close()
-
-	// Creates a client to execute tests against the test server
-	client, err := api.New(ts.URL)
-	require.NoError(t, err)
-
-	out, err := client.UserCreate(context.Background(), &api.User{})
-	require.NoError(t, err)
-	require.Equal(t, fixture.ID, out.ID)
-	require.Equal(t, fixture.UserName, out.UserName)
-}
-
-func TestUserDetail(t *testing.T) {
-	fixture := &api.User{
-		ID:       "001",
-		UserName: "username01",
+func TestTenantList(t *testing.T) {
+	fixture := &api.TenantPage{
+		Tenants: []*api.Tenant{
+			{
+				ID:              "001",
+				TenantName:      "tenant01",
+				EnvironmentType: "Dev",
+			},
+			{
+				ID:              "002",
+				TenantName:      "tenant02",
+				EnvironmentType: "Prod",
+			},
+			{
+				ID:              "003",
+				TenantName:      "tenant03",
+				EnvironmentType: "Stage",
+			},
+		},
+		PrevPageToken: "2121",
+		NextPageToken: "4040",
 	}
 
 	// Creates a test server
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		require.Equal(t, http.MethodGet, r.Method)
-		require.Equal(t, "/v1/users/:id", r.URL.Path)
+		require.Equal(t, "/v1/tenant", r.URL.Path)
+		require.Equal(t, "next_page_token=&page_size=0", r.URL.RawQuery)
```

Comment:
This is an ok test for now, but I just wanted to alert you to the possibility that it is a bit fragile since there is no guarantee that `next_page_token` will always come before `page_size`.  A more structured approach would be to parse the `r.URL.RawQuery` into a `url.Values` and use `require.Contains` or `require.Equal` on the parsed map to do the test.